man/systemd.service.xml

```diff
@@ -1331,6 +1331,9 @@ WantedBy=multi-user.target</programlisting>
       <varname>ExecStart=</varname> specified. They will be executed
       in order until either they are all successful or one of them
       fails.</para>
+      <para><varname>Restart=</varname> can accept <option>on-failure</option>
+      <option>on-abnormal</option> <option>on-watchdog</option> <option>on-abort</option>.
```

Comment:
~~This is missing commas in between the words.~~

Scratch that. This text is in the wrong place. Here, you should just say:
`For units with multiple commands (<varname noindex="true">Type=oneshot</varname>), all command will be run again.`

And in the description of Restart=, say
`For <varname noindex="true">Type=oneshot</varname>, <option>always</> and <option>on-success</> are <emphasis>not</> allowed.`